otter/convergence/model.py

```diff
@@ -126,6 +154,69 @@ def __init__(self):
         assert self.state in ServerState.iterconstants(), \
             "%r is not a ServerState" % (self.state,)
 
+    @classmethod
+    def group_id_from_metadata(cls, metadata):
```

Comment:
Ok yeah, since we're undecided on whether NovaServer should have properties or attributes, I'll leave them here for now and then maybe move them.  :|